asmcomp/cmmgen.ml

```diff
@@ -542,7 +545,7 @@ let get_tag ptr dbg =
   if Proc.word_addressed then           (* If byte loads are slow *)
     Cop(Cand, [get_header ptr dbg; Cconst_int 255], dbg)
   else                                  (* If byte loads are efficient *)
-    Cop(Cload Byte_unsigned,
+    Cop(Cload (Byte_unsigned, Mutable), (* Same comment as above *)
```

Comment:
Not obvious what this comment refers to. Maybe just explicitly repeat that tags can be mutated by `Obj.set_tag`.